frontend/src/components/Form.js

```diff
@@ -229,6 +228,16 @@ function Form() {
   }
 
   function handleChange(event) {
+    if (recipe.user === "") {
+      console.log("handle change is fixign user");
+      const currentUser = AuthService.getCurrentUser();
+      console.log(currentUser.username);
+      let another = recipe;
+      let name = String(currentUser.username);
+      another["user"] = name;
+      setRecipe(another);
```

Comment:
Is it possible to assign username to the recipe by calling setRecipe("user": currentUser.username)?

Reply:
Yes, that was my original attempt but it was buggy and didn't always work. This seems to do the trick